vms/components/fee/dimensions_test.go

```diff
@@ -426,3 +426,44 @@ func Test_Dimensions_ToGas(t *testing.T) {
 		})
 	}
 }
+
+func Benchmark_Dimensions_Add(b *testing.B) {
+	require := require.New(b)
+
+	lhs := Dimensions{600, 10, 10, 1000}
+	rhs := []Dimensions{
+		{1, 1, 1, 1},
+		{10, 10, 10, 10},
+		{100, 100, 100, 100},
+		{200, 200, 200, 200},
+		{500, 500, 500, 500},
+		{1_000, 1_000, 1_000, 1_000},
+		{10_000, 10_000, 10_000, 10_000},
+	}
+
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		_, err := lhs.Add(rhs...)
```

Comment:
By passing in varargs with a slice expansion, you are not copying the dimensions. The slice is passed in directly without copying.